deployment/refinery-modules/refinery/manifests/init.pp

```diff
@@ -82,7 +82,7 @@
   apt::source { 'nodejs':
     ensure      => 'present',
     comment     => 'Nodesource NodeJS repo.',
-    location    => 'https://deb.nodesource.com/node_6.x',
+    location    => 'https://deb.nodesource.com/node_8.x',
```

Comment:
This should be thoroughly tested before merging into develop:
* both on AWS (launch stacks with and without snapshots) and Vagrant (provision existing and brand new VMs)
* deployment logs should be examined for any errors or warnings in all cases
* at least the home page should be loaded and browser console should be checked for any errors or warnings in all cases